test/fixtures/Bed.json

```diff
@@ -0,0 +1,47 @@
+[
```

Comment:
we've 3 centre fixtures it'd make sense to have more in the way of fixtures just to prove that no one makes a silly mistake like a truthy evaluation of a centre id to `centre.id == 1`